docs/data/material/getting-started/templates/marketing-page/components/FAQ.tsx

```diff
@@ -61,15 +64,15 @@ export default function FAQ() {
               gutterBottom
               sx={{ maxWidth: { sm: '100%', md: '70%' } }}
             >
-              You can reach our customer support team by emailing
-              <Link> support@email.com </Link>
-              or calling our toll-free number. We&apos;re here to assist you
+              You can reach our customer support team by emailing&nbsp;
+              <Link href="mailto:support@email.com">support@email.com</Link>
+              &nbsp;or calling our toll-free number. We&apos;re here to assist you
```

Comment:
This looked broken:

<img width="288" alt="SCR-20241118-uiqa" src="https://github.com/user-attachments/assets/e4687a31-824a-4894-9596-4c4638eca597">